thorlcr/master/thgraphmanager.cpp

```diff
@@ -1545,6 +1630,8 @@ void thorMain(ILogMsgHandler *logHandler, const char *wuid, const char *graphNam
                                 break; // timeout/abort
                             // else - reject/ignore duff message.
                         }
+                        while (!lingerTimer.timedout(&lingerRemaining));
```

Comment:
style: it is arguably clearer that this belongs with the do {} if it is on the same line as the }

Reply:
am not sure about that, feel contrary to rest of blocking style IMO, but also not that fussed, will change.